diff -up chromium-125.0.6422.60/components/variations/service/ui_string_overrider.cc.default-constructor-involving-anonymous-union chromium-125.0.6422.60/components/variations/service/ui_string_overrider.cc
--- chromium-125.0.6422.60/components/variations/service/ui_string_overrider.cc.default-constructor-involving-anonymous-union	2024-05-15 23:46:17.000000000 +0200
+++ chromium-125.0.6422.60/components/variations/service/ui_string_overrider.cc	2024-05-17 12:33:58.684649909 +0200
@@ -12,7 +12,7 @@
 
 namespace variations {
 
-UIStringOverrider::UIStringOverrider() = default;
+UIStringOverrider::UIStringOverrider() {}
 
 UIStringOverrider::UIStringOverrider(base::span<const uint32_t> resource_hashes,
                                      base::span<const int> resource_indices)
diff -up chromium-125.0.6422.60/content/browser/interest_group/header_direct_from_seller_signals.cc.default-constructor-involving-anonymous-union chromium-125.0.6422.60/content/browser/interest_group/header_direct_from_seller_signals.cc
--- chromium-125.0.6422.60/content/browser/interest_group/header_direct_from_seller_signals.cc.default-constructor-involving-anonymous-union	2024-05-15 23:46:17.000000000 +0200
+++ chromium-125.0.6422.60/content/browser/interest_group/header_direct_from_seller_signals.cc	2024-05-17 12:33:58.685649929 +0200
@@ -46,7 +46,7 @@ size_t GetResultSizeBytes(const HeaderDi
 
 }  // namespace
 
-HeaderDirectFromSellerSignals::Result::Result() = default;
+HeaderDirectFromSellerSignals::Result::Result() {}
 
 HeaderDirectFromSellerSignals::Result::Result(
     std::optional<std::string> seller_signals,
diff -up chromium-125.0.6422.60/third_party/blink/renderer/bindings/core/v8/frozen_array.h.default-constructor-involving-anonymous-union chromium-125.0.6422.60/third_party/blink/renderer/bindings/core/v8/frozen_array.h
--- chromium-125.0.6422.60/third_party/blink/renderer/bindings/core/v8/frozen_array.h.default-constructor-involving-anonymous-union	2024-05-18 09:52:22.385330304 +0200
+++ chromium-125.0.6422.60/third_party/blink/renderer/bindings/core/v8/frozen_array.h	2024-05-18 09:52:51.023891525 +0200
@@ -36,7 +36,7 @@ class FrozenArray final : public binding
   using const_iterator = typename VectorType::const_iterator;
   using const_reverse_iterator = typename VectorType::const_reverse_iterator;
 
-  FrozenArray() = default;
+  FrozenArray() {}
   explicit FrozenArray(VectorType array) : array_(std::move(array)) {}
   ~FrozenArray() override = default;
 
