diff -up chromium-126.0.6478.26/chrome/renderer/accessibility/read_anything_app_model.cc.stdformat chromium-126.0.6478.26/chrome/renderer/accessibility/read_anything_app_model.cc
--- chromium-126.0.6478.26/chrome/renderer/accessibility/read_anything_app_model.cc.stdformat	2024-06-03 22:52:18.596936767 +0200
+++ chromium-126.0.6478.26/chrome/renderer/accessibility/read_anything_app_model.cc	2024-06-03 23:26:46.024734453 +0200
@@ -1059,7 +1059,7 @@ std::string ReadAnythingAppModel::GetHtm
     int32_t hierarchical_level =
         ax_node->GetIntAttribute(ax::mojom::IntAttribute::kHierarchicalLevel);
     if (hierarchical_level) {
-      return std::format("h{}", hierarchical_level);
+      return std::string("h" + std::to_string(hierarchical_level));
     }
   }
 
@@ -1159,7 +1159,7 @@ std::string ReadAnythingAppModel::GetHea
   int32_t hierarchical_level =
       ax_node->GetIntAttribute(ax::mojom::IntAttribute::kHierarchicalLevel);
   if (hierarchical_level) {
-    return std::format("h{}", hierarchical_level);
+    return std::string("h" + std::to_string(hierarchical_level));
   }
   return html_tag;
 }
