diff -up chromium-124.0.6367.49/build/config/compiler/BUILD.gn.orig chromium-124.0.6367.49/build/config/compiler/BUILD.gn
--- chromium-124.0.6367.49/build/config/compiler/BUILD.gn.orig	2024-04-14 22:43:02.906578948 +0200
+++ chromium-124.0.6367.49/build/config/compiler/BUILD.gn	2024-04-14 22:46:19.513353809 +0200
@@ -1890,9 +1890,6 @@ config("default_warnings") {
 
         # TODO(crbug.com/1494809): Evaluate and possibly enable.
         "-Wno-vla-extension",
-
-        # TODO(https://crbug.com/1490607): Fix and re-enable.
-        "-Wno-thread-safety-reference-return",
       ]
 
       cflags_cc += [
@@ -1903,7 +1900,7 @@ config("default_warnings") {
       if (!is_nacl) {
         cflags_cc += [
           # TODO(https://crbug.com/1513724): Fix and re-enable.
-          "-Wno-c++11-narrowing-const-reference",
+          "-Wno-c++11-narrowing",
         ]
       }
     }
