diff -up chromium-120.0.6099.35/base/allocator/partition_allocator/src/partition_alloc/starscan/metadata_allocator.h.me chromium-120.0.6099.35/base/allocator/partition_allocator/src/partition_alloc/starscan/metadata_allocator.h
--- chromium-120.0.6099.35/base/allocator/partition_allocator/src/partition_alloc/starscan/metadata_allocator.h.me	2023-11-26 13:33:11.547409713 +0100
+++ chromium-120.0.6099.35/base/allocator/partition_allocator/src/partition_alloc/starscan/metadata_allocator.h	2023-11-26 13:35:02.679363167 +0100
@@ -40,6 +40,11 @@ class MetadataAllocator {
   }
 
   template <typename U>
+  bool operator==(const MetadataAllocator<U>&) const {
+    return true;
+  }
+
+  template <typename U>
   bool operator!=(const MetadataAllocator<U>& o) {
     return !operator==(o);
   }
diff -up chromium-120.0.6099.35/third_party/pdfium/core/fxcrt/fx_memory_wrappers.h.me chromium-120.0.6099.35/third_party/pdfium/core/fxcrt/fx_memory_wrappers.h
--- chromium-120.0.6099.35/third_party/pdfium/core/fxcrt/fx_memory_wrappers.h.me	2023-11-22 20:34:11.000000000 +0100
+++ chromium-120.0.6099.35/third_party/pdfium/core/fxcrt/fx_memory_wrappers.h	2023-11-26 13:30:38.961724653 +0100
@@ -82,6 +82,7 @@ struct FxPartitionAllocAllocator {
   }
 
   // There's no state, so they are all the same,
+  bool operator==(const FxPartitionAllocAllocator&) const { return true; }
   bool operator==(const FxPartitionAllocAllocator& that) { return true; }
   bool operator!=(const FxPartitionAllocAllocator& that) { return false; }
 };
