diff -up chromium-121.0.6167.57/base/allocator/partition_allocator/src/partition_alloc/partition_alloc_config.h.me chromium-121.0.6167.57/base/allocator/partition_allocator/src/partition_alloc/partition_alloc_config.h
--- chromium-121.0.6167.57/base/allocator/partition_allocator/src/partition_alloc/partition_alloc_config.h.me	2024-01-16 14:20:52.401890657 +0100
+++ chromium-121.0.6167.57/base/allocator/partition_allocator/src/partition_alloc/partition_alloc_config.h	2024-01-16 15:08:04.070156474 +0100
@@ -162,7 +162,8 @@ static_assert(sizeof(void*) != 8, "");
 
 #if defined(ARCH_CPU_ARM64) && defined(__clang__) && \
     !defined(ADDRESS_SANITIZER) &&                   \
-    (BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_ANDROID))
+    (BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_ANDROID)) && \
+    __GLIBC_PREREQ(2, 29)
 #define PA_CONFIG_HAS_MEMORY_TAGGING() 1
 #else
 #define PA_CONFIG_HAS_MEMORY_TAGGING() 0
