diff -up chromium-121.0.6167.57/media/ffmpeg/ffmpeg_common.cc.me chromium-121.0.6167.57/media/ffmpeg/ffmpeg_common.cc
--- chromium-121.0.6167.57/media/ffmpeg/ffmpeg_common.cc.me	2024-01-16 10:54:38.994173911 +0100
+++ chromium-121.0.6167.57/media/ffmpeg/ffmpeg_common.cc	2024-01-16 11:06:20.974160612 +0100
@@ -710,8 +710,13 @@ bool AVStreamToVideoDecoderConfig(const
   }
 
   VideoTransformation video_transformation = VideoTransformation();
+#if LIBAVCODEC_VERSION_INT < AV_VERSION_INT(60, 31, 102)
+  for (int i = 0; i < stream->nb_side_data; i++) {
+    const auto& side_data = stream->side_data[i];
+#else
   for (int i = 0; i < stream->codecpar->nb_coded_side_data; ++i) {
     const auto& side_data = stream->codecpar->coded_side_data[i];
+#endif
     switch (side_data.type) {
       case AV_PKT_DATA_DISPLAYMATRIX: {
         CHECK_EQ(side_data.size, sizeof(int32_t) * 3 * 3);
diff -up chromium-121.0.6167.57/media/filters/audio_video_metadata_extractor.cc.me chromium-121.0.6167.57/media/filters/audio_video_metadata_extractor.cc
--- chromium-121.0.6167.57/media/filters/audio_video_metadata_extractor.cc.me	2024-01-16 11:12:27.521534151 +0100
+++ chromium-121.0.6167.57/media/filters/audio_video_metadata_extractor.cc	2024-01-16 11:15:08.717337026 +0100
@@ -113,8 +113,13 @@ bool AudioVideoMetadataExtractor::Extrac
     if (!stream)
       continue;
 
+#if LIBAVCODEC_VERSION_INT < AV_VERSION_INT(60, 31, 102)
+    for (int j = 0; j < stream->nb_side_data; j++) {
+      const AVPacketSideData& sd = stream->side_data[j];
+#else
     for (int j = 0; j < stream->codecpar->nb_coded_side_data; j++) {
       const AVPacketSideData& sd = stream->codecpar->coded_side_data[j];
+#endif
       if (sd.type == AV_PKT_DATA_DISPLAYMATRIX) {
         CHECK_EQ(sd.size, sizeof(int32_t) * 3 * 3);
         rotation_ = VideoTransformation::FromFFmpegDisplayMatrix(
