diff -up chromium-119.0.6045.105/chrome/browser/media/router/media_router_feature.cc.disable-GlobalMediaControlsCastStartStop chromium-119.0.6045.105/chrome/browser/media/router/media_router_feature.cc
--- chromium-119.0.6045.105/chrome/browser/media/router/media_router_feature.cc.disable-GlobalMediaControlsCastStartStop	2023-11-06 12:57:04.096696491 +0100
+++ chromium-119.0.6045.105/chrome/browser/media/router/media_router_feature.cc	2023-11-06 13:00:30.480926343 +0100
@@ -68,15 +68,9 @@ BASE_FEATURE(kCastMirroringPlayoutDelay,
              base::FEATURE_DISABLED_BY_DEFAULT);
 const base::FeatureParam<int> kCastMirroringPlayoutDelayMs{
     &kCastMirroringPlayoutDelay, "cast_mirroring_playout_delay_ms", -1};
-#if BUILDFLAG(IS_CHROMEOS)
 BASE_FEATURE(kGlobalMediaControlsCastStartStop,
              "GlobalMediaControlsCastStartStop",
              base::FEATURE_DISABLED_BY_DEFAULT);
-#else
-BASE_FEATURE(kGlobalMediaControlsCastStartStop,
-             "GlobalMediaControlsCastStartStop",
-             base::FEATURE_ENABLED_BY_DEFAULT);
-#endif  // BUILDFLAG(IS_CHROMEOS)
 #endif  // !BUILDFLAG(IS_ANDROID)
 
 namespace {
