diff -up chromium-125.0.6422.60/chrome/browser/ui/webui/commerce/shopping_insights_side_panel_ui.h.const chromium-125.0.6422.60/chrome/browser/ui/webui/commerce/shopping_insights_side_panel_ui.h
--- chromium-125.0.6422.60/chrome/browser/ui/webui/commerce/shopping_insights_side_panel_ui.h.const	2024-05-18 18:57:37.099203314 +0200
+++ chromium-125.0.6422.60/chrome/browser/ui/webui/commerce/shopping_insights_side_panel_ui.h	2024-05-18 22:16:52.879603051 +0200
@@ -42,7 +42,7 @@ class ShoppingInsightsSidePanelUI
       mojo::PendingReceiver<
           shopping_service::mojom::ShoppingServiceHandlerFactory> receiver);
 
-  static constexpr std::string GetWebUIName() {
+  static const std::string GetWebUIName() {
     return "ShoppingInsightsSidePanel";
   }
 
diff -up chromium-125.0.6422.60/chrome/browser/ui/webui/compose/compose_untrusted_ui.h.const chromium-125.0.6422.60/chrome/browser/ui/webui/compose/compose_untrusted_ui.h
--- chromium-125.0.6422.60/chrome/browser/ui/webui/compose/compose_untrusted_ui.h.const	2024-05-18 19:03:13.999255471 +0200
+++ chromium-125.0.6422.60/chrome/browser/ui/webui/compose/compose_untrusted_ui.h	2024-05-18 22:16:52.883603124 +0200
@@ -58,7 +58,7 @@ class ComposeUntrustedUI
     triggering_web_contents_ = web_contents->GetWeakPtr();
   }
 
-  static constexpr std::string GetWebUIName() { return "Compose"; }
+  static const std::string GetWebUIName() { return "Compose"; }
 
  private:
   void CreateComposeSessionUntrustedPageHandler(
diff -up chromium-125.0.6422.60/chrome/browser/ui/webui/lens/lens_untrusted_ui.h.const chromium-125.0.6422.60/chrome/browser/ui/webui/lens/lens_untrusted_ui.h
--- chromium-125.0.6422.60/chrome/browser/ui/webui/lens/lens_untrusted_ui.h.const	2024-05-18 18:59:16.859995432 +0200
+++ chromium-125.0.6422.60/chrome/browser/ui/webui/lens/lens_untrusted_ui.h	2024-05-18 22:16:52.883603124 +0200
@@ -47,7 +47,7 @@ class LensUntrustedUI : public Untrusted
       mojo::PendingReceiver<color_change_listener::mojom::PageHandler>
           receiver);
 
-  static constexpr std::string GetWebUIName() { return "LensUntrusted"; }
+  static const std::string GetWebUIName() { return "LensUntrusted"; }
 
  private:
   // lens::mojom::LensPageHandlerFactory:
diff -up chromium-125.0.6422.60/chrome/browser/ui/webui/lens/search_bubble_ui.h.const chromium-125.0.6422.60/chrome/browser/ui/webui/lens/search_bubble_ui.h
--- chromium-125.0.6422.60/chrome/browser/ui/webui/lens/search_bubble_ui.h.const	2024-05-18 18:59:49.020573177 +0200
+++ chromium-125.0.6422.60/chrome/browser/ui/webui/lens/search_bubble_ui.h	2024-05-18 22:16:52.883603124 +0200
@@ -26,7 +26,7 @@ class SearchBubbleUI : public TopChromeW
       mojo::PendingReceiver<lens::mojom::SearchBubblePageHandlerFactory>
           receiver);
 
-  static constexpr std::string GetWebUIName() { return "LensSearchBubble"; }
+  static const std::string GetWebUIName() { return "LensSearchBubble"; }
 
  private:
   // lens::mojom::SearchBubblePageHandlerFactory
diff -up chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/bookmarks/bookmarks_side_panel_ui.h.const chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/bookmarks/bookmarks_side_panel_ui.h
--- chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/bookmarks/bookmarks_side_panel_ui.h.const	2024-05-18 19:00:08.686926471 +0200
+++ chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/bookmarks/bookmarks_side_panel_ui.h	2024-05-18 22:16:52.883603124 +0200
@@ -63,7 +63,7 @@ class BookmarksSidePanelUI
   commerce::ShoppingListContextMenuController*
   GetShoppingListContextMenuController();
 
-  static constexpr std::string GetWebUIName() { return "BookmarksSidePanel"; }
+  static const std::string GetWebUIName() { return "BookmarksSidePanel"; }
 
  private:
   // side_panel::mojom::BookmarksPageHandlerFactory:
diff -up chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/companion/companion_side_panel_untrusted_ui.h.const chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/companion/companion_side_panel_untrusted_ui.h
--- chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/companion/companion_side_panel_untrusted_ui.h.const	2024-05-18 19:00:36.351423440 +0200
+++ chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/companion/companion_side_panel_untrusted_ui.h	2024-05-18 22:16:52.884603143 +0200
@@ -34,7 +34,7 @@ class CompanionSidePanelUntrustedUI
   // Gets a weak pointer to this object.
   base::WeakPtr<CompanionSidePanelUntrustedUI> GetWeakPtr();
 
-  static constexpr std::string GetWebUIName() {
+  static const std::string GetWebUIName() {
     return "CompanionSidePanelUntrusted";
   }
 
diff -up chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/customize_chrome/customize_chrome_ui.h.const chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/customize_chrome/customize_chrome_ui.h
--- chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/customize_chrome/customize_chrome_ui.h.const	2024-05-18 19:00:56.088778012 +0200
+++ chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/customize_chrome/customize_chrome_ui.h	2024-05-18 22:16:52.884603143 +0200
@@ -102,7 +102,7 @@ class CustomizeChromeUI
           side_panel::customize_chrome::mojom::WallpaperSearchHandlerFactory>
           pending_receiver);
 
-  static constexpr std::string GetWebUIName() { return "CustomizeChrome"; }
+  static const std::string GetWebUIName() { return "CustomizeChrome"; }
 
  private:
   // side_panel::mojom::CustomizeChromePageHandlerFactory
diff -up chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/history_clusters/history_clusters_side_panel_ui.h.const chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/history_clusters/history_clusters_side_panel_ui.h
--- chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/history_clusters/history_clusters_side_panel_ui.h.const	2024-05-18 19:01:15.986135446 +0200
+++ chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/history_clusters/history_clusters_side_panel_ui.h	2024-05-18 22:16:52.884603143 +0200
@@ -70,7 +70,7 @@ class HistoryClustersSidePanelUI : publi
   void DidFinishNavigation(
       content::NavigationHandle* navigation_handle) override;
 
-  static constexpr std::string GetWebUIName() {
+  static const std::string GetWebUIName() {
     return "HistoryClustersSidePanel";
   }
 
diff -up chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/performance_controls/performance_side_panel_ui.h.const chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/performance_controls/performance_side_panel_ui.h
--- chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/performance_controls/performance_side_panel_ui.h.const	2024-05-18 19:01:34.274463982 +0200
+++ chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/performance_controls/performance_side_panel_ui.h	2024-05-18 22:16:52.884603143 +0200
@@ -41,7 +41,7 @@ class PerformanceSidePanelUI
       mojo::PendingReceiver<color_change_listener::mojom::PageHandler>
           pending_receiver);
 
-  static constexpr std::string GetWebUIName() { return "PerformanceSidePanel"; }
+  static const std::string GetWebUIName() { return "PerformanceSidePanel"; }
 
  private:
   // side_panel::mojom::PerformancePageHandlerFactory:
diff -up chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/read_anything/read_anything_untrusted_ui.h.const chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/read_anything/read_anything_untrusted_ui.h
--- chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/read_anything/read_anything_untrusted_ui.h.const	2024-05-18 19:01:52.842797556 +0200
+++ chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/read_anything/read_anything_untrusted_ui.h	2024-05-18 22:16:52.884603143 +0200
@@ -55,7 +55,7 @@ class ReadAnythingUntrustedUI
       mojo::PendingReceiver<read_anything::mojom::UntrustedPageHandlerFactory>
           receiver);
 
-  static constexpr std::string GetWebUIName() {
+  static const std::string GetWebUIName() {
     return "ReadAnythingUntrusted";
   }
 
diff -up chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/reading_list/reading_list_ui.h.const chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/reading_list/reading_list_ui.h
--- chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/reading_list/reading_list_ui.h.const	2024-05-18 19:02:17.381238375 +0200
+++ chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/reading_list/reading_list_ui.h	2024-05-18 22:16:52.885603162 +0200
@@ -47,7 +47,7 @@ class ReadingListUI : public TopChromeWe
 
   void SetActiveTabURL(const GURL& url);
 
-  static constexpr std::string GetWebUIName() { return "ReadingList"; }
+  static const std::string GetWebUIName() { return "ReadingList"; }
 
  private:
   // reading_list::mojom::PageHandlerFactory:
diff -up chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/user_notes/user_notes_side_panel_ui.h.const chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/user_notes/user_notes_side_panel_ui.h
--- chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/user_notes/user_notes_side_panel_ui.h.const	2024-05-18 19:02:34.413544341 +0200
+++ chromium-125.0.6422.60/chrome/browser/ui/webui/side_panel/user_notes/user_notes_side_panel_ui.h	2024-05-18 22:16:52.885603162 +0200
@@ -35,7 +35,7 @@ class UserNotesSidePanelUI
 
   void set_browser(Browser* browser) { browser_ = browser; }
 
-  static constexpr std::string GetWebUIName() { return "UserNotesSidePanel"; }
+  static const std::string GetWebUIName() { return "UserNotesSidePanel"; }
 
  private:
   // user_notes::mojom::UserNotesPageHandlerFactory
diff -up chromium-125.0.6422.60/chrome/browser/ui/webui/tab_search/tab_search_ui.h.const chromium-125.0.6422.60/chrome/browser/ui/webui/tab_search/tab_search_ui.h
--- chromium-125.0.6422.60/chrome/browser/ui/webui/tab_search/tab_search_ui.h.const	2024-05-18 19:02:52.261864965 +0200
+++ chromium-125.0.6422.60/chrome/browser/ui/webui/tab_search/tab_search_ui.h	2024-05-18 22:16:52.885603162 +0200
@@ -46,7 +46,7 @@ class TabSearchUI : public TopChromeWebU
     return page_handler_.get();
   }
 
-  static constexpr std::string GetWebUIName() { return "TabSearch"; }
+  static const std::string GetWebUIName() { return "TabSearch"; }
 
  private:
   // tab_search::mojom::PageHandlerFactory
diff -up chromium-125.0.6422.60/components/omnibox/browser/on_device_tail_model_service.cc.const chromium-125.0.6422.60/components/omnibox/browser/on_device_tail_model_service.cc
--- chromium-125.0.6422.60/components/omnibox/browser/on_device_tail_model_service.cc.const	2024-05-18 23:06:22.071586909 +0200
+++ chromium-125.0.6422.60/components/omnibox/browser/on_device_tail_model_service.cc	2024-05-18 23:06:44.827133461 +0200
@@ -30,7 +30,7 @@ namespace {
 // The maximum idle time before the model executor is unloaded from memory.
 constexpr base::TimeDelta kMaxExecutorIdleSeconds = base::Seconds(60);
 // Constants for TFlite model validation.
-constexpr std::string kTestPrefix = "face";
+const std::string kTestPrefix = "face";
 constexpr std::string_view kModelValidationSwitchName =
     "omnibox-on-device-tail-model-validation";
 
diff -up chromium-124.0.6367.118/components/autofill/core/browser/data_model/autofill_i18n_api.h.me chromium-124.0.6367.118/components/autofill/core/browser/data_model/autofill_i18n_api.h
--- chromium-124.0.6367.118/components/autofill/core/browser/data_model/autofill_i18n_api.h.me	2024-05-07 07:58:10.202960298 +0200
+++ chromium-124.0.6367.118/components/autofill/core/browser/data_model/autofill_i18n_api.h	2024-05-07 07:59:29.413411767 +0200
@@ -16,8 +16,8 @@ namespace autofill::i18n_model_definitio
 // Country code that represents autofill's legacy address hierarchy model as
 // stored `kAutofillModelRules`. As a workaround for GCC we declare the
 // std::string constexpr first.
-constexpr inline std::string kLegacyHierarchyCountryCodeString{"XX"};
-constexpr AddressCountryCode kLegacyHierarchyCountryCode =
+const inline std::string kLegacyHierarchyCountryCodeString{"XX"};
+const AddressCountryCode kLegacyHierarchyCountryCode =
     AddressCountryCode(kLegacyHierarchyCountryCodeString);
 
 // Creates an instance of the address hierarchy model corresponding to the
diff -up chromium-124.0.6367.118/components/autofill/core/browser/metrics/log_event.h.me chromium-124.0.6367.118/components/autofill/core/browser/metrics/log_event.h
--- chromium-124.0.6367.118/components/autofill/core/browser/metrics/log_event.h.me	2024-05-07 07:59:38.239573497 +0200
+++ chromium-124.0.6367.118/components/autofill/core/browser/metrics/log_event.h	2024-05-07 08:00:55.578990659 +0200
@@ -57,7 +57,7 @@ struct IsRequired {
   // This function is not defined and consteval. Therefore, any evaluation will
   // fail and fail at compile time.
   template <typename T>
-  consteval operator T();  // NOLINT
+  operator T();  // NOLINT
 };
 
 }  // namespace internal
diff -up chromium-124.0.6367.118/components/history/core/browser/history_types.h.me chromium-124.0.6367.118/components/history/core/browser/history_types.h
--- chromium-124.0.6367.118/components/history/core/browser/history_types.h.me	2024-05-06 18:01:28.606845554 +0200
+++ chromium-124.0.6367.118/components/history/core/browser/history_types.h	2024-05-06 18:06:15.492751252 +0200
@@ -70,7 +70,7 @@ constexpr VisitedLinkID kInvalidVisitedL
 typedef std::map<VisitID, VisitSource> VisitSourceMap;
 
 // Constant used to represent that no app_id is used for matching.
-inline constexpr std::optional<std::string> kNoAppIdFilter = std::nullopt;
+inline const std::optional<std::string> kNoAppIdFilter = std::nullopt;
 
 // VisitRow -------------------------------------------------------------------
 
diff -up chromium-124.0.6367.118/services/network/shared_dictionary/simple_url_pattern_matcher.cc.me chromium-124.0.6367.118/services/network/shared_dictionary/simple_url_pattern_matcher.cc
--- chromium-124.0.6367.118/services/network/shared_dictionary/simple_url_pattern_matcher.cc.me	2024-05-07 00:07:15.720573794 +0200
+++ chromium-124.0.6367.118/services/network/shared_dictionary/simple_url_pattern_matcher.cc	2024-05-07 00:08:07.633519604 +0200
@@ -28,17 +28,17 @@ namespace network {
 namespace {
 
 // https://urlpattern.spec.whatwg.org/#default-options
-constexpr liburlpattern::Options kDefaultOptions = {.delimiter_list = "",
+const liburlpattern::Options kDefaultOptions = {.delimiter_list = "",
                                                     .prefix_list = "",
                                                     .sensitive = true,
                                                     .strict = true};
 // https://urlpattern.spec.whatwg.org/#hostname-options
-constexpr liburlpattern::Options kHostnameOptions = {.delimiter_list = ".",
+const liburlpattern::Options kHostnameOptions = {.delimiter_list = ".",
                                                      .prefix_list = "",
                                                      .sensitive = true,
                                                      .strict = true};
 // https://urlpattern.spec.whatwg.org/#pathname-options
-constexpr liburlpattern::Options kPathnameOptions = {.delimiter_list = "/",
+const liburlpattern::Options kPathnameOptions = {.delimiter_list = "/",
                                                      .prefix_list = "/",
                                                      .sensitive = true,
                                                      .strict = true};
diff -up chromium-124.0.6367.118/components/autofill/core/browser/form_parsing/transpile_regex_patterns.py.me1 chromium-124.0.6367.118/components/autofill/core/browser/form_parsing/transpile_regex_patterns.py
--- chromium-124.0.6367.118/components/autofill/core/browser/form_parsing/transpile_regex_patterns.py.me1	2024-05-07 10:58:13.316756926 +0200
+++ chromium-124.0.6367.118/components/autofill/core/browser/form_parsing/transpile_regex_patterns.py	2024-05-07 10:58:30.821083061 +0200
@@ -54,7 +54,7 @@ def generate_matching_pattern_equals(nam
   yield '// Checks if all the matching patterns for the given PatternSources'
   yield '// and language are the same - meaning that computing predictions for'
   yield '// both is unnecessary, since it will yield the same result.'
-  yield 'constexpr bool AreMatchingPatternsEqualImpl(PatternSource a,'
+  yield 'const bool AreMatchingPatternsEqualImpl(PatternSource a,'
   yield '                                            PatternSource b,'
   yield '                                            LanguageCode lang_code) {'
   yield '  if (a == b) {'
