diff -up chromium-115.0.5790.102/mojo/public/tools/mojom/mojom/fileutil.py.me chromium-115.0.5790.102/mojo/public/tools/mojom/mojom/fileutil.py
--- chromium-115.0.5790.102/mojo/public/tools/mojom/mojom/fileutil.py.me	2023-07-22 13:48:01.726074705 +0200
+++ chromium-115.0.5790.102/mojo/public/tools/mojom/mojom/fileutil.py	2023-07-22 13:50:50.886575507 +0200
@@ -3,7 +3,6 @@
 # found in the LICENSE file.
 
 import errno
-import imp
 import os.path
 import sys
 
diff -up chromium-115.0.5790.102/mojo/public/tools/mojom/mojom/parse/lexer.py.me chromium-115.0.5790.102/mojo/public/tools/mojom/mojom/parse/lexer.py
--- chromium-115.0.5790.102/mojo/public/tools/mojom/mojom/parse/lexer.py.me	2023-07-22 14:10:40.372979365 +0200
+++ chromium-115.0.5790.102/mojo/public/tools/mojom/mojom/parse/lexer.py	2023-07-22 14:13:31.200453442 +0200
@@ -2,7 +2,6 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-import imp
 import os.path
 import sys
 
