diff -up chromium-116.0.5845.50/base/allocator/partition_allocator/partition_alloc_config.h.me chromium-116.0.5845.50/base/allocator/partition_allocator/partition_alloc_config.h
--- chromium-116.0.5845.50/base/allocator/partition_allocator/partition_alloc_config.h.me	2023-07-30 15:13:45.873427874 +0200
+++ chromium-116.0.5845.50/base/allocator/partition_allocator/partition_alloc_config.h	2023-07-30 15:15:31.393181400 +0200
@@ -150,9 +150,7 @@ static_assert(sizeof(void*) != 8, "");
   (!BUILDFLAG(PUT_REF_COUNT_IN_PREVIOUS_SLOT) && \
    defined(ARCH_CPU_LITTLE_ENDIAN))
 
-#define PA_CONFIG_HAS_MEMORY_TAGGING()              \
-  (defined(ARCH_CPU_ARM64) && defined(__clang__) && \
-   (BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_ANDROID)))
+#define PA_CONFIG_HAS_MEMORY_TAGGING() 0
 
 #if PA_CONFIG(HAS_MEMORY_TAGGING)
 static_assert(sizeof(void*) == 8);
