diff -up third_party/ffmpeg/ffmpeg_generated.gni.me third_party/ffmpeg/ffmpeg_generated.gni
--- third_party/ffmpeg/ffmpeg_generated.gni.me	2024-05-05 00:45:53.017100841 +0200
+++ third_party/ffmpeg/ffmpeg_generated.gni	2024-05-05 00:46:20.244559951 +0200
@@ -17,8 +17,8 @@ use_linux_config = is_linux || is_chrome
 if ((is_android && current_cpu == "arm" && arm_use_neon) ||
     (is_android && current_cpu == "arm64") ||
     (is_android && current_cpu == "x64") ||
-    (is_android && current_cpu == "x86") || is_apple || is_win ||
-    use_linux_config) {
+    (is_android && current_cpu == "x86") || (is_apple) || (is_win) ||
+    (use_linux_config)) {
   ffmpeg_c_sources += [
     "libavcodec/ac3_channel_layout_tab.c",
     "libavcodec/ac3_parser.c",
